PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/RootLayoutController.java

```diff
@@ -162,4 +162,13 @@ public void filesLoaded(InputStream treeFile, InputStream graphFile, InputStream
       Logger.getLogger(getClass().getName()).log(Level.SEVERE, null, e);
     }
   }
+
+  @FXML
+  @SuppressWarnings("unused")
+  private void openFileMenuItemClicked() throws IOException {
```

Comment:
Maybe we can call this method from [here](https://github.com/ProgrammingLife2016/PL2-2016/blob/feat/open-file-dialog/PL2/PL2-launcher/src/main/java/nl/tudelft/pl2016gr2/launcher/Dnav.java#L45)?

With this the SupressWarnings can go too.